app/models/review.rb

```diff
@@ -0,0 +1,11 @@
+class Review < ApplicationRecord
+  belongs_to :park
+
+  validates :title, presence: true
+  validates :body, presence: true
+  validates :rating, presence: true, numericality: {
+    greater_than_or_equal_to: 1,
+    less_than_or_equal_to: 5,
+    only_integer: true  
+  }
+end
```

Comment:
Review model tests - validations and associations

Reply:
Thanks! We've added a new model test for Review below.